server/src/main/java/org/elasticsearch/index/engine/Engine.java

```diff
@@ -228,8 +231,42 @@ protected final DocsStats docsStats(IndexReader indexReader) {
             } catch (IOException e) {
                 logger.trace(() -> "failed to get size for [" + info.info.name + "]", e);
             }
+            docsWithIgnoredFields = getValueOrZero(
```

Comment:
Like we discussed earlier, before when we computed `DocStats` we didn't get and read metadata for an inverted index. This in general is cheap, but this will be done for each shard in the cluster, some of these shards maybe less powerful hardware. The `DocStats` is included by default, so we in large clusters with cold tier, we run at risk of increasing the latency of indices and cluster stats API calls.

Maybe instead we should introduce a new stat group (`IgnoredFieldStats`?) that isn't enabled by default?